services/QuillLMS/client/app/bundles/Teacher/components/classrooms/import_google_classroom_students_modal.tsx

```diff
@@ -64,35 +61,54 @@ export default class ImportGoogleClassroomStudentsModal extends React.Component<
     return buttonClass;
   }
 
-  toggleCheckbox() {
-    const newStateObj = { checkboxOne: !this.state.checkboxOne, }
+  handleToggleCheckbox() {
+    const { checkboxOne } = this.state
+    const newStateObj = { checkboxOne: !checkboxOne }
     this.setState(newStateObj)
   }
 
   renderCheckbox() {
-    const checkbox = this.state.checkboxOne
-    if (checkbox) {
-      return <div className="quill-checkbox selected" onClick={this.toggleCheckbox}><img alt="check" src={smallWhiteCheckSrc} /></div>
+    const { checkboxOne } = this.state
+
+    if (checkboxOne) {
+      return (
+        <div className="quill-checkbox selected" onClick={this.handleToggleCheckbox}>
+          <img alt="check" src={smallWhiteCheckSrc} />
+        </div>
+      )
     } else {
-      return <div className="quill-checkbox unselected" onClick={this.toggleCheckbox} />
+      return <div className="quill-checkbox unselected" onClick={this.handleToggleCheckbox} />
     }
   }
 
   renderCheckboxes() {
-    return (<div className="checkboxes">
-      <div className="checkbox-row">
-        {this.renderCheckbox()}
-        <span>I understand that newly imported students have access to the activities that have already been assigned to the entire class.</span>
+    return (
+      <div className="checkboxes">
+        <div className="checkbox-row">
+          {this.renderCheckbox()}
+          <span>
+            I understand that newly imported students have access to the activities that have already been assigned
```

Comment:
I think this is where the `to` got removed (but shouldn't have been) that I called out in the test file.